src/main/java/com/aa_code/online_food_ordering_backend/Model/User.java

```diff
@@ -24,6 +25,8 @@ public class User {
 
     private String fullName;
     private String email;
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
when the response came password was not required to the response section. That is why I disabled it from the @JsonProperty annotation. There was a bug in the signup section without the access part. Because when a user registers it password section ignores the JSON input. So i fix the the bug with the access part.